portfolio/src/main/webapp/scripts/listenersMain.js

```diff
@@ -0,0 +1,9 @@
+const options = document.querySelectorAll(".menu-option");
+const factButton = document.querySelector("#fact-button");
+
+for(let i=0; i<options.length; i++){
+  options[i].addEventListener('click', function(){toggleOverlay(options[i].id)})
```

Comment:
For readability, put blocks on their own lines, indented by 2 spaces.

options[i].addEventListener('click', function(){
  toggleOverlay(options[i].id)
})

Same for the line below